CONTRIBUTING.md

```diff
@@ -0,0 +1,203 @@
+
+### Compiling and Testing Changes
+
+When you are ready to test any changes you make, compile **Nucleus Coop** by opening its solution file and running a batch build on it within Visual Studio ("Master\NucleusCoop.sln"). Similar to Proto Input, on the Batch Build menu, select all the x86 and x64 options for the build you would like to compile. 
+
+Once you have selected your build options and run the batch build command, your output build directory should now have a fully functioning versio of the program and everything it needs to run! To test your changes live within Nucleus Coop itself, go to your build directory and run "NucleusCoop.exe" to test out your new build.
+
+As a reminder, please take care to follow the Coding Standards for the project when adding any code-based contributions.
+
+Test your changes **thoroughly** before submitting a Pull Request. Whenever approrpriate, it is strongly recomended that you develop unit-tests with a good test spread to make sure that your new features are working as intended and are capable of handling anticipated edge cases. In cases where this is not possible, please be sure to follow a similar testing paradigm where both average and edge use cases are tested. In the long run, this will save the project a lot of tedious back-tracking and bug-smashing by proactively ensuring that easily-preventable bugs don't sneak into the master build.
+
+#### Build Configurations
+* Release - The release version of the app (live, user-facing build)
+* Debug - Debug version of the app (extra debug features enabled)
+
+### Submitting Changes
+
+#### Create a Pull Request
+Once you have finished building and testing your changes, it's time to submit them! From the /SplitScreen-Me/splitscreenme-nucleus repository, go to the Pull Requests section and click the "New Pull Request" button.
+
+**Make sure that the _base_ repository is set to "/SplitScreen-Me/splitscreenme-nucleus"!** 
+
+Otherwise, you will be attempting to merge your changes with a different repository. Double-check that you will be merging with the correct head branch as well (generally "master") Also verify that the _head_ repository is set to your fork and the branch of your fork where you made all the changes that you would like to implement.
+
+It is a good idea to create a detailed (but concise) comment to accompany the initial pull request to explain:
+* The issue the pull request is addressing
+* What is changed
+* Testing procedures used and their results
+* Potential issues or challenges encountered
+* Any questions or other comments for contribution reviewers
+
+#### Review Process
+All pull requests will approved by a sufficient number of project maintainers (those with write access to the project repository) before any changes are merged. Before aproving changes, reviewers will determine whether:

```

Comment:
I'm incorporating this suggestion manually, since it needs to go along with a slight tweak to phrasing of one or two of the following bullet points - will be implemented in the next uploaded commit.